### PR TITLE
Added a mist reaction

### PR DESCRIPTION
**Pull request by PolterTzi**
## About the PR

Adds a mist reaction - equivalent to smoke, but doesn't obstruct vision and has a much more transparent sprite.
The recipe is 1:1:1 Sodium, Phosphorus and Sugar, but that's honestly just kinda arbitrary, absolutely open to changing that.

## Why / Balance

The intent is to enable chemical grenades that are able to infuse people with chems without otherwise impeding gameplay - Either for funny purposes, medical application or somewhat more stealthy poisonings.

## Media

https://github.com/space-wizards/space-station-14/assets/62557990/0ad7dbf1-8cc7-4369-bfbd-9806158ee56d

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: Mixing Sugar, Sodium and Phosphorus now creates a transparent chemical mist.

### CODE REVIEW

**Comment by TheShuEd** [comment]
Atmos refactor is here :trollface:

**Comment by UbaserB** [comment]
If the chem injection is on the same scale as foam/fog, and THIS easy to craft, i feel like that is unbalanced and there is no reason to use either one. I would either increase the recipe or reduce the effects.

**Comment by PolterTzi** [comment]
> If the chem injection is on the same scale as foam/fog, and THIS easy to craft, i feel like that is unbalanced and there is no reason to use either one. I would either increase the difficulty of the recipe or reduce the effects.

If your goal is solely to inject chems, it is meant to be a better option than either.
The two still would have their uses, since sometimes you still do want to cover up sightlines or slip people or apply the chem to the floor.

Maybe it's too hard to notice with the current sprite, I can see why that would be an issue. That could be tweaked.
And I'm more intent to trying to add this as a viable option than this exact recipe, so, yeah, it could be tweaked.

File: Resources/Prototypes/Entities/Effects/chemistry_effects.yml [hunk 1]
```diff
@@ -47,6 +47,21 @@
     tags:
     - HideContextMenu
 
+- type: entity
+  parent: BaseFoam
+  id: Mist
+  name: mist
+  noSpawn: true
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Effects/chemistry_effects.yml:54]: This field is obsolete.

**Review by metalgearsloth — CHANGES_REQUESTED** [review]
Isn't this just easy to craft smoke / foam

**Comment by PolterTzi** [comment]
> Isn't this just easy to craft smoke / foam

I wanted something that would achieve a similar effect to smoke without making everyone affected effectively blind.
I don't think it's that much easier to make than smoke, but if the current recipe is unbalanced, it can be made a bit harder.

File: Resources/Prototypes/Entities/Effects/chemistry_effects.yml (hunk 1 above)
Comment (PolterTzi) [reply Resources/Prototypes/Entities/Effects/chemistry_effects.yml:54]: I assume you mean noSpawn?

**Comment by Jezithyr** [comment]
Reopened since the repoban was in error. (The ban was made due to a misunderstood reference to the stonewall riots in an april fools PR)

**Comment by Dueu** [comment]
My proposal for balancing: Space Aerosol Reagent.
Recipe: 3 Oil, 2 Hydrogen, 4 Flourine, Catalyst: Silver OR Vestine = 9 Space Aerosol
This locks it behind asking cargo for silver or being a chemist agent and keep vestine to make space aerosol.
Then it would be a reaction 1 water to 1 space aerosol plus the reagents to turn into mist.

**Comment by mirrorcult** [comment]
this is overly confusing in its role when placed alongside foam and smoke, hard to notice and i dont understand why there necessarily needs to be a reagent like smoke that doesnt obscure vision

**Pull request closed without merging**